modyn/supervisor/internal/eval/strategies/_interval.py

```diff
@@ -66,4 +57,4 @@ def get_eval_intervals(
                 + int(self.config.right) * SECONDS_PER_UNIT[self.config.right_unit]
                 - int(not self.config.right_bound_inclusive)
             )
-        yield max(0, left), max(left, right) if right else right
+        return max(0, left), max(left, right) if right else right
```

Comment:
can this ever return None (according to the type specification it can, but we init left and right with 0 and apply max operations on them)